src/optimizer/join_order_optimizer.cpp

```diff
@@ -338,15 +329,14 @@ JoinNode *JoinOrderOptimizer::EmitPair(JoinRelationSet *left, JoinRelationSet *r
 #ifdef DEBUG
```

Comment:
Could you move the symmetry verification to another method? Could be a static method or a private method of the JoinOrderOptimizer.
```c++
void VerifySymmetry(args ...) {
#ifdef DEBUG
    // ... verification code
#endif
}
```

This keeps the code for `CreateJoinTree` more clean.